src/app/pages/forms-and-validation/phone-number-format/shared/format-phone-number.pipe.ts

```diff
@@ -0,0 +1,54 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'formatPhoneNumber',
+})
+export class FormatPhoneNumberPipe implements PipeTransform {
+    transform(value: string, country: string, countryChange?: boolean): string {
+        if (countryChange) {
+            return value;
+        }
+        let phoneNumber = value.replace(/\s/g, '');
+        switch (country) {
+            case 'US':
+            case 'CA': {
+                if (phoneNumber.length > 3 && phoneNumber.length <= 6)
+                    phoneNumber = `${phoneNumber.slice(0, 3)} ${phoneNumber.slice(3)}`;
+                else if (phoneNumber.length > 6)
+                    phoneNumber = `${phoneNumber.slice(0, 3)} ${phoneNumber.slice(3, 6)} ${phoneNumber.slice(6)}`;
+                return phoneNumber;
+            }
```

Comment:
Let's clean this up a bit by moving these cases down with the default case since it's the same.

Reply:
I was hoping to remove lines 15-20 and adding the `case 'US': case 'CA':` just above the `default:` like I explained in the other switch example.